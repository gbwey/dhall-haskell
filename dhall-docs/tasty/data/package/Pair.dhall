
{-|
Just a *Pair* right here
-}

let Pair
  : forall(A: Type) -- | Type of first position
  -> forall(B: Type) -- | Type of second position
  -> Type
  = λ(A: Type) -> λ(B: Type) -> {first: A, second: B}

in Pair
