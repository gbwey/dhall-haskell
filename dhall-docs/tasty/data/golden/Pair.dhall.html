<!DOCTYPE HTML>
<html
  ><head
    ><title
    >/Pair.dhall</title
    ><link href="index.css" type="text/css" rel="stylesheet"
    /><link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap"
            type="text/css" rel="stylesheet"
    /><script src="index.js" type="text/javascript"
    /><meta charset="UTF-8"/></head
  ><body
    ><div class="nav-bar"
      ><img src="dhall-icon.svg" class="dhall-icon"
      /><p class="package-title"
      >test-package</p
      ><div class="nav-bar-content-divider"
      /><a id="switch-light-dark-mode" class="nav-option"
      >Switch Light/Dark Mode</a></div
    ><div class="main-container"
      ><h2 class="doc-title"
        ><span class="crumb-divider"
        >/</span
        ><a href="index.html"
        >test-package</a
        ><span class="crumb-divider"
        >/</span
        ><span href="index.html" class="title-crumb"
        >Pair.dhall</span></h2
      ><a data-path="/Pair.dhall" class="copy-to-clipboard"
        ><i
          ><small
          >Copy path to clipboard</small></i></a
      ><br
      /><div class="doc-contents"
        ><p
          >Just a <em
          >Pair</em> right here</p>
</div
      ><h3
      >Source</h3
      ><div class="source-code"
        ><pre
          ><br
          />{-|<br
          />Just a *Pair* right here<br
          />-}<br
          /><br
          />let Pair<br
          />  : forall(A: Type) -- | Type of first position<br
          />  -&gt; forall(B: Type) -- | Type of second position<br
          />  -&gt; Type<br
          />  = λ(A: Type) -&gt; λ(B: Type) -&gt; {first: A, second: B}<br
          /><br
          />in Pair<br/></pre></div></div></body></html>